src/aws-provider/aws-organization-writer.ts

```diff
@@ -38,14 +39,16 @@ export class AwsOrganizationWriter {
         }
     }
 
-    public async ensureSCPEnabled(): Promise<void> {
+    public async ensureSCPEnabled(mirror: boolean): Promise<void> {
         return await performAndRetryIfNeeded(async () => {
+            const org: Organizations = (mirror) ? this.partitionOrgService : this.organizationService;
```

Comment:
if, on line 37, we assign the "partitionOrgService" to "this.organizationService" we dont have to repeat the `mirror ? this.partitionedService: this.commercialService`